parm/archive/gefs_extracted_wave.yaml.j2

```
@@ -0,0 +1,41 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + "wave.t" + cycle_HH + "z." %}
+
+gefs_wave:
+    name: "GEFS_WAVE"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_wave.tar"
+    required:
+{% if REPLAY_ICS %}
+    {% set ofst_hr = OFFSET_START_HOUR %}
+{% else %}
+    {% set ofst_hr = FHMIN_GFS %}
+{% endif %}
+
+{% set res = (waveGRD[-3:])[0] ~ "p" ~ (waveGRD[-3:])[-2:] %}
+
+#select mem%03d and ensstat files required
+{% set members = [] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do members.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+{% for mem in members %}
+    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                          '${RUN}':RUN,
+                          '${YMD}':cycle_YMD,
+                          '${HH}':cycle_HH,
+                          '${MEMDIR}':mem }) %}
+
+    {% set COMIN_WAVE_GRID = COM_WAVE_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+    # Select grib2 files to copy to the atardir
+    {% if path_exists(COMIN_WAVE_GRID) %}
+        {% for fhr in range(ofst_hr, FHMAX_GFS + FHOUT_HF_WAV, FHOUT_HF_WAV) %}
```

We may want to use `FHOUT_WAV` here. 
```suggestion
        {% for fhr in range(ofst_hr, FHMAX_GFS + FHOUT_WAV, FHOUT_WAV) %}
```